packages/office-ui-fabric-react/src/components/SearchBox/SearchBox.scss

```diff
@@ -38,12 +83,12 @@ $SearchBox-height: 36px;
 .rootIsActive {
   border-color: $ms-color-themeDarker;
 
-  .field {
-    @include padding-left(8px);
-  }
-
   .icon {
-    display: none;
+    flex-basis: 4px;
+    transition: flex-basis $ms-duration1;
+     & > * {
+      display: none;
```

Comment:
the search icon is hidden when the search is active. by setting display to none I can then reduce the flex basis of the parent and animate the change.